### DISCUSSION TITLE
CF Roadmap - Schedule a meeting

### DISCUSSION THREAD

**Discussion by ethanrd** [cf-convention/discuss#386] [Announcements relevant to CF]
Hi all - Sorry I'm late with this task. Let's find a time to have a meeting to review and plan for the CF Roadmap. I have put together a poll for 14:00 or 15:00 UTC over the next two weeks. Please fill it out here (https://whenisgood.net/zq8j34a) if you are interested in attending.

Pinging - @lhmarsden @gtesoro @erget @sadielbartholomew @feggleton @efisher008 @agstephens @sethmcg @DocOtak @davidhassell @japamment @sebvi @JonathanGregory @larsbarring @pagecp @chris-little @cofinoa

**Comment by ethanrd** [comment]
Sorry all. Just realized that timezones shift out of daylight savings in the next few weeks depending on where you are. So the UTC times I gave are not correct for the second week. Sorry for any confusion that might cause.

**Comment by ethanrd** [comment]
Hi all - Let's go with 4 November at 15:00 UTC (8am US/Mtn, 3pm BT, 4pm/CET). I'll send an email with a calendar invite and Google Meet link to everyone that responded. If you are interested in attending but didn't respond, let me know (you can find my email on my GH page).

**Comment by efisher008** [comment]
[like]  Fisher, Ellie (STFC,RAL,RALSP) reacted to your message:
________________________________
